src/system.h

```diff
@@ -61,6 +61,7 @@ struct db_filter_col;
 #define SECCOMP_RET_TRAP	0x00030000U /* disallow and force a SIGSYS */
 #define SECCOMP_RET_ERRNO	0x00050000U /* returns an errno */
 #define SECCOMP_RET_TRACE	0x7ff00000U /* pass to a tracer or disallow */
+#define SECCOMP_RET_LOG		0x7ffc0000U /* allow after logging */
```

Comment:
We added SECCOMP_RET_LOG back in 3b22b153c70ef00044fb309d9a9fecc12b777e98, does that commit not work for you?

Reply:
I suppose we could add this bit of the patch, but drop the section below as commit 3b22b153 should have that covered.

Reply:
This duality between system.h and seccomp.h.in is a little bit overengineered, IMO.

However, I'm going to import SECCOMP_RET_USER_NOTIF into system.h (for testing that the return type is supported) in the user-notification series, so I think it makes sense to add this one here as well, to keep things in sync with the kernel headers. Although yes, it's a no-op for this series.